SPECS/ucx/ucx.spec

```diff
@@ -310,56 +394,61 @@ library internals, protocol objects, transports status, and more.
 %{_bindir}/ucx_vfs
 %endif
 
+%if %{with ze}
+%package ze
+Requires: %{name} = %{version}-%{release}
+Summary: UCX ZE GPU support
+Group: System Environment/Libraries
+
+%description ze
+Provides oneAPI Level Zero (ZE) Runtime support for UCX.
+
+%files ze
+%{_libdir}/ucx/libuct_ze.so.*
+%{_libdir}/ucx/libucm_ze.so.*
+%endif
+
 %changelog
-* Fri Jan 26 2024 Juan Camposeco <juanarturoc@microsoft.com> - 1.15.0-5
+* Tue Nov 26 2024 Alberto David Perez Guevara <aperezguevar@microsoft.com> 1.18.0-3
+- Azure Linux update to version 1.18.0
+* Thu Nov 07 2024 Suresh Babu Chalamalasetty <schalam@microsoft.com> 1.18.0-2
+- Initial version Azure Linux
+* Fri Apr 19 2024 Yossi Itigin <yosefe@nvidia.com> 1.18.0-1
+- Bump version to 1.18.0
+* Tue Oct 31 2023 Yossi Itigin <yosefe@nvidia.com> 1.17.0-1
+- Bump version to 1.17.0
+* Fri Apr 28 2023 Yossi Itigin <yosefe@nvidia.com> 1.16.0-1
+- Bump version to 1.16.0
+* Mon Oct 24 2022 Yossi Itigin <yosefe@mellanox.com> 1.15.0-1
+- Bump version to 1.15.0
+* Sat Apr 16 2022 Yossi Itigin <yosefe@mellanox.com> 1.14.0-1
+- Bump version to 1.14.0
+* Wed Nov 10 2021 Yossi Itigin <yosefe@mellanox.com> 1.13.0-1
+- Bump version to 1.13.0
+* Wed Jun 9 2021 Yossi Itigin <yosefe@mellanox.com> 1.12.0-1
+- Bump version to 1.12.0
+* Fri Jan 26 2024 Juan Camposeco <juanarturoc@microsoft.com> 1.15.0-5
 - Update version to 1.15.0 and remove knem dependency
-
-* Wed Sep 20 2023 Jon Slobodzian <joslobo@microsoft.com> - 1.11.0-4
+* Wed Sep 20 2023 Jon Slobodzian <joslobo@microsoft.com> 1.11.0-4
 - Recompile with stack-protection fixed gcc version (CVE-2023-4039)
-
-* Tue Jul 26 2022 Rachel Menge <rachelmenge@microsoft.com> - 1.11.0-3
+* Tue Jul 26 2022 Rachael Menge <rachelmenge@microsoft.com> 1.11.0-3
 - Move from SPECS-EXTENDED to SPECS
 - License verified
-
-* Thu Aug 26 2021 Thomas Crain <thcrain@microsoft.com> - 1.11.0-2
+* Thu Aug 26 2021 Thomas Crain <thcrain@microsoft.com> 1.11.0-2
 - Initial CBL-Mariner import from Fedora 35 (license: MIT).
 - Remove option to build with libibcm
```

Comment:
There's a lot of changes in the changelog and we now have conflicting information. There's this line saying we got the spec from Fedora and Suresh's line above saying it's our own version of the spec. If we switched to our own version, please clean-up the spec to mention the appropriate source.

Reply:
Ok will clean up this